app/src/sharedTest/java/org/oppia/app/topic/train/TopicTrainFragmentTest.kt

```diff
@@ -26,9 +31,13 @@ import javax.inject.Singleton
 class TopicTrainFragmentTest {
 
   @Test
```

Comment:
Please include KDoc for test cases

Reply:
Is there specific documentation you're looking for @nikitamarysolomanpvt? Typically we don't document tests (mostly because they should be sufficiently self-explanatory), but sometimes it's useful.

Reply:
Initially I was using autogenerated test case method names in android studio.I had gone through your slide on test case then understood the new approach, the approach is quite nice and as per that we could understand the test and expected result by the method name. I think we could just give the link of the standards (ppt which explain this) that we are following inside doc (for new contributors  to understand)and avoid self-explanatory method KDoc.